CMakeLists.txt

```diff
@@ -31,18 +32,21 @@ ProcessorCount(NumProcessors)
 message("Number of processors: ${NumProcessors}")
 file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/nprocs.txt ${NumProcessors})
 
```

Comment:
Not sure if this should be built this way or if the external parser should be standalone and built from inside the `external_parser` directory